cg/services/order_validation_service/workflows/tomte/validation/inter_field_validators.py

```diff
@@ -0,0 +1,42 @@
+from cg.services.order_validation_service.models.validation_error import (
+    ValidationError,
+    ValidationErrors,
+)
+from cg.services.order_validation_service.workflows.tomte.models.case import TomteCase
+from cg.services.order_validation_service.workflows.tomte.models.order import TomteOrder
+
+
+def each_case_contains_mother(order: TomteOrder) -> ValidationErrors:
```

Comment:
I wonder whether these can be implemented as Field validators directly on the Pydantic model so we levarage those features a bit more? Or is there merit to split it up like this?

Reply:
We've decided to keep interfield validation outside of the Pydantic models because it makes the models messy